### PR TITLE
fix(swap): fix swap lag

### PR DESCRIPTION
**Pull request by plondon**
## Description (optional)
fix swap lagginesssssss

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/modals/Swap/CoinSelection/index.tsx [hunk 1]
```diff
@@ -38,52 +36,40 @@ class CoinSelection extends PureComponent<Props> {
...
-      (side === 'BASE' &&
-        values?.COUNTER?.type === SwapBaseCounterTypes.ACCOUNT &&
-        account.type === SwapBaseCounterTypes.CUSTODIAL)
-    ) {
-      return true
-    }
-    return false
-  }
+  render() {
+    // @ts-ignore
```
Comment (cmonti-bc) [inline packages/blockchain-wallet-v4-frontend/src/modals/Swap/CoinSelection/index.tsx:40]: Is this ok?

File: packages/blockchain-wallet-v4-frontend/src/modals/Swap/CoinSelection/selectors.ts [hunk 2]
```diff
@@ -14,13 +18,58 @@ export const getData = (state: RootState, { side }: OwnProps) => {
...
   const pairs = selectors.components.swap.getPairs(state).getOrElse([])
-  let coinsForSide
+  const values = selectors.form.getFormValues('initSwap')(state) as InitSwapFormValuesType
+  const custodialEligibility = selectors.components.swap
+    .getCustodialEligibility(state)
+    .getOrElse(false)
 
   if (!pairs.length) return { accounts }
 
+  const _checkBaseCustodial = (account: SwapAccountType) => {
```
Comment (cmonti-bc) [inline packages/blockchain-wallet-v4-frontend/src/modals/Swap/CoinSelection/selectors.ts:28]: is this method name a convention for the internal or private methods?

**Review by blockdylanb — APPROVED** [review]
Looks good. We should implement react-window on other areas where we show coin lists as well!

File: packages/blockchain-wallet-v4-frontend/src/modals/Swap/CoinSelection/selectors.ts (hunk 2 above)
Comment (plondon) [reply packages/blockchain-wallet-v4-frontend/src/modals/Swap/CoinSelection/selectors.ts:28]: Not a convention just wanted to make it clearer that these are helper methods inside the selector

File: packages/blockchain-wallet-v4-frontend/src/modals/Swap/CoinSelection/index.tsx (hunk 1 above)
Comment (plondon) [reply packages/blockchain-wallet-v4-frontend/src/modals/Swap/CoinSelection/index.tsx:40]: Hmm good spot, dont think I need this anymore

**Review by rsteubs-bc — APPROVED** [review]

**Review by blockdylanb — APPROVED** [review]

**Pull request merged**